db/migrate/migration_utils/legacy_yamler.rb

```diff
@@ -32,33 +32,10 @@
 module Migration
   module LegacyYamler
     ##
-    # Tries to load syck and fails with an error
-    # if it was not installed.
-    # To continue with the affected migrations, install syck with `bundle install --with syck`
+    # Loads the given yaml with syck.
+    # This may be altered to allow an optional require in the future.
     def load_with_syck(yaml)
-      @@syck ||= load_syck
-      @@syck.load(yaml)
-    end
-
-    private
-
-    def load_syck
-      require 'syck'
-      ::Syck
-    rescue LoadError => e
-      abort = -> (str) { abort("\e[31m#{str}\e[0m") }
-      abort.call <<-WARN
-      It appears you have existing serialized YAML in your database.
-
-      This YAML may have been serialized with Syck, which allowed to parse YAML
-      that is now considered invalid given the default Ruby YAML parser (Psych),
-      we need to convert that YAML to be Psych-compatible.
-
-      Use `bundle install --with syck` to install the syck YAML parser
-      and re-run the migrations.
-      WARN
-
-      raise e
+      ::Syck.load(yaml)
```

Comment:
Always writing this and that... bad habit...

I think it is a good thing to have `require: false` set and thus needing to require Syck manually.

I'd even think about leaving this file as it was and just adding another comment like "will currently not cause load errors because of our Gemfile"...